src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -217,6 +220,7 @@ export function Autosuggest<S>(props: Props<S>) {
             selectOnTab
             initInputValue={initInputValue}
             clearInputAfterSelection
+            indexToSelectOnOutClick={selectFirstOnOutClick ? 0 : -1}
```

Comment:
Should we use  `highlightedIndex` instead of the first item (`0`)?

Reply:
after discussion we decided that it always be the first item